apps/api/src/report/report.service.ts

```diff
@@ -221,37 +221,149 @@ export class ReportService {
    */
   async splitReportFourNewOldProcess(f: string, t: string) {
     const statuses = await this.getStatusMap();
-    const entityManager = getManager();
-
     const { from, to } = this.captureFromTo(f, t);
 
     this.logger.log(`getLicensingStageApplicants: Apply date filter from (${from}) and to (${to})`);
+    const connection = getConnection();
+    const mappedStatuses = LIC_REG_STAGE.map(licensingStatus => {
+      return statuses[licensingStatus];
+    });
+    const mappedStatusesString = mappedStatuses.map(status => `'${status}'`).join(',');
```

Comment:
could we do this in one line? as in tack on .join to the `mappedStatuses` map and just wrap the return like ``'${statuses[licensingStatus]}'`